autoPyTorch/api/base_task.py

```diff
@@ -170,13 +170,13 @@ def __init__(
             os.path.join(os.path.dirname(__file__), '../configs/default_pipeline_options.json'))))
 
         self.search_space: Optional[ConfigurationSpace] = None
-        self._dataset_requirements: Optional[List[FitRequirement]] = None
         self._metric: Optional[autoPyTorchMetric] = None
         self._logger: Optional[PicklableClientLogger] = None
         self.run_history: RunHistory = RunHistory()
         self.trajectory: Optional[List] = None
-        self.dataset_name: Optional[str] = None
+        self.dataset_name: str = ""
```

Comment:
why do you want to have it as an empty string?

Reply:
Okay, I see that it was suggested by Francisco in a previous PR. I suggest you to not make the same changes in different PR. Let merge #106 first and then we can rebase to include this change. It makes it confusing to review the same changes in different places.